### PR TITLE
Api 18q1 updates

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: hxntools/anc350.py [hunk 1]
```diff
@@ -41,30 +41,30 @@ def setup_dc(self, enable, period, off_time, verify=True):
...
-            set_and_wait(self.dc_enable, enable)
+            yield from bps.wait(group=wait_group)
 
 
 class HxnAnc350Axis(Anc350Axis):
     def __init__(self, controller, axis_num, **kwargs):
         prefix = 'XF:03IDC-ES{{ANC350:{}-Ax:{}}}'.format(controller, axis_num)
-        super().__init__(prefix, axis_num=axis_num, **kwargs)
+        name = f'anc_axis_{controller}'
+        super().__init__(prefix, axis_num=axis_num, name=name, **kwargs)
```
Comment (licode) [inline hxntools/anc350.py:60]: we also need to define name attribute in Anc350Axis and Anc350Controller?

**Comment by licode** [comment]
LGTM. Another question in general: are we going to move detector like zebra to ophyd later? So it is easy to synchronize.

File: hxntools/anc350.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def _wait_tries(signal, value, tries=20, period=0.1):
 
 def _dc_toggle(axis, enable, freq, dc_period, off_time):
     print('Axis {} {}: '.format(axis.axis_num, axis.desc.value), end='')
-    set_and_wait(axis.frequency, freq)
+    yield from bps.mov(axis.frequency, freq)
```
Comment (mrakitin) [inline hxntools/anc350.py:95]: Should it be `bps.mv`?

**Review by mrakitin — COMMENTED** [review]
Looks good to me, except of the minor issue with `bps.mov`.

File: hxntools/anc350.py (hunk 2 above)
Comment (tacaswell) [reply hxntools/anc350.py:95]: yes, but we might provide both

**Comment by tacaswell** [comment]
There is one more `signal_names` -> `component_names` left on this branch.

File: hxntools/anc350.py (hunk 2 above)
Comment (danielballan) [reply hxntools/anc350.py:95]: Indeed. We will maintain both forever because no one can agree what which name is better.

**Pull request merged**